recipe/meta.yaml

```diff
@@ -40,14 +41,18 @@ requirements:
     - dask-core >=0.15
     - toolz >=0.7.4
     - pywavelets >=0.4.0
-    - dask-core >=0.9.0
```

Comment:
Why are you requiring 0.15 on conda but 0.9 on pip?

Reply:
a) none of these changes are mine. I had to fix some merge conflicts between your commits and master, and master was already on 0.15.
b) I suspect that the combination of Python 3.6 / dask-core only exists in certain versions, so it's probably not a huge point?
c) does dask-core < 0.15 even exist on conda? Actually, no, it doesn't, just checked.

Reply:
a) I wonder who made those changes.

Reply:
The truth is out there. =P